doc/cascadia/profiles.schema.json

```
@@ -0,0 +1,505 @@
+          },
+          "guid": {
+            "$ref": "#/definitions/ProfileGuid",
+            "description": "Unique identifier of the profile. Written in registry format: \"{00000000-0000-0000-0000-000000000000}\"."
+          },
+          "historySize": {
+            "default": 9001,
+            "description": "The number of lines above the ones displayed in the window you can scroll back to.",
+            "minimum": -1,
+            "type": "integer"
+          },
+          "icon": {
+            "description": "Image file location of the icon used in the profile. Displays within the tab and the dropdown menu.",
+            "type": "string"
+          },
+          "name": {
+            "description": "Name of the profile. Displays in the dropdown menu.",
+            "minLength": 1,
+            "type": "string"
+          },
+          "padding": {
+            "default": "8, 8, 8, 8",
+            "description": "Sets the padding around the text within the window. Can have three different formats: \"#\" sets the same padding for all sides, \"#, #\" sets the same padding for left-right and top-bottom, and \"#, #, #, #\" sets the padding individually for left, top, right, and bottom.",
+            "pattern": "^-?[0-9]+(\\.[0-9]+)?( *, *-?[0-9]+(\\.[0-9]+)?|( *, *-?[0-9]+(\\.[0-9]+)?){3})?$",
+            "type": "string"
+          },
+          "scrollbarState": {
+            "default": "visible",
+            "description": "Defines the visibility of the scrollbar.",
+            "enum": [
+              "visible",
+              "hidden"
+            ],
+            "type": "string"
+          },
+          "snapOnInput": {
+            "default": true,
+            "description": "When set to true, the window will scroll to the command input line when typing. When set to false, the window will not scroll when you start typing.",
+            "type": "boolean"
+          },
+          "source": {
+            "description": "Stores the name of the profile generator that originated this profile.",
+            "type": "string"
+          },
+          "startingDirectory": {
+            "description": "The directory the shell starts in when it is loaded.",
+            "type": "string"
+          },
+          "tabTitle": {
+            "description": "If set, will replace the name as the title to pass to the shell on startup. Some shells (like bash) may choose to ignore this initial value, while others (cmd, powershell) may use this value over the lifetime of the application.",
+            "type": "string"
+          },
+          "useAcrylic": {
+            "default": false,
+            "description": "When set to true, the window will have an acrylic background. When set to false, the window will have a plain, untextured background.",
+            "type": "boolean"
+          }
```

The "hidden" property is missing.
```suggestion
          },
          "hidden": {
            "default": false,
            "description": "If set to true, the profile will not appear in the list of profiles. This can be used to hide default profiles and dynamicially generated profiles, while leaving them in your settings file.",
            "type": "boolean"
          }
```